.k8s/deploy.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
```

Comment:
any reason it's `.k8s` as opposed to just `.k8s`? what's the convention here?

Reply:
`.[folder name]/` makes the folder hidden in linux, it's usually there to... encourage that non-related build processes should ignore that folder just in case there are operations that do things with the entire source folder, kind of like `.git` and `.github`.  that and i like to hide folders that don't directly deal with the program code itself

Reply:
and shitty IDEs like eclipse sometimes pukes up a bunch of warnings saying "boo hoo this doesn't look like javascript why is it here" as well